pycparser/c_ast.py

```diff
@@ -76,8 +73,16 @@ def show(self, buf=sys.stdout, offset=0, attrnames=False, nodenames=False, showc
             showcoord:
                 Do you want the coordinates of each Node to be
                 displayed.
+
+            indent:
+                Spaces and lines to be displayed behind node.
```

Comment:
I don't understand what this means -- "displayed behind node"?

Reply:
Left over from older version, forgot to change. As for the failing tests I accidentally typed an 's' near the top of the file